.github/workflows/CI.yml

```diff
@@ -103,6 +103,11 @@ jobs:
           conda remove --force openff-toolkit openff-toolkit-base
           python setup.py develop --no-deps
 
+      - name: Install test plugins
+        run: |
+          cd utilities/test_plugins
+          python setup.py develop --no-deps
+
```

Comment:
Does the next action default to starting out in the root directory? I used to do a lot of `cd x/y/ && do thing && cd ../../`, which was maybe wrong all along. Later steps seem to run fine, so I guess that answer is "yes"

Reply:
That's.... a really good question. I'm also going to assume the answer is "yes", for the same reasons as you said.